src/gsy_e/models/strategy/scm/load.py

```diff
@@ -0,0 +1,63 @@
+from typing import Dict, TYPE_CHECKING
+
+from pendulum import DateTime
+
+from gsy_e.models.strategy.load_hours import LoadHoursPerDayEnergyParameters
+from gsy_e.models.strategy.predefined_load import DefinedLoadEnergyParameters
+from gsy_e.models.strategy.scm import SCMStrategy
+
+if TYPE_CHECKING:
+    from gsy_e.models.area import AreaBase
+
+
+class SCMLoadHoursStrategy(SCMStrategy):
+    """Load SCM strategy with constant power production."""
+    def __init__(self, avg_power_W, hrs_per_day=None, hrs_of_day=None):
+        self._energy_params = LoadHoursPerDayEnergyParameters(avg_power_W, hrs_per_day, hrs_of_day)
+        self._simulation_start_timestamp = None
+
+    def activate(self, area: "AreaBase") -> None:
+        """Activate the strategy."""
+        self._energy_params.event_activate_energy(area)
+        self._simulation_start_timestamp = area.now
+
+    def _get_day_of_timestamp(self, time_slot: DateTime) -> int:
+        """Return the number of days passed from the simulation start date to the time slot."""
+        if self._simulation_start_timestamp is None:
+            return 0
+        return (time_slot - self._simulation_start_timestamp).days
+
+    def market_cycle(self, area: "AreaBase") -> None:
+        """Update the load forecast and measurements for the next/previous market slot."""
+        # self._update_energy_requirement_in_state()
+        # # Provide energy values for the past market slot, to be used in the settlement market
+        # self._set_energy_measurement_of_last_market()
+
+    def get_available_energy_kWh(self, time_slot: DateTime) -> float:
+        """Get the available energy for consumption for the specified time slot."""
+        return self._energy_params.state.get_energy_requirement_Wh(time_slot) / 1000.0
```

Comment:
I'm a bit confused by this, probably because of the naming. The method is about getting the "available energy", but it's returning the "required energy" of the load. Was this made to simplify the interface? I find it a bit "conceptually" convoluted.

Reply:
(This is not currently called anywhere, right?)

Reply:
Indeed this was an effort to simplify the interface. Ideally I want in the followup story for the SCM clearing algorithm, to harmonize the interface so that I can do from the Area class this: `self.strategy.get_energy_to_buy_kWh(time_slot)`

no matter the strategy class of the area. get_available_energy_kWh is probably a bad name for this though, so I would recommend either the following:
```
def get_energy_to_sell_kWh()
def get_energy_to_buy_kWh()
```

or this:
```
def get_energy_production_kWh()
def get_energy_consumption_kWh()
```

I tend to like the former more, however let me know if you prefer the latter and I will harmonise the interface even now.

Reply:
And WRT your second comment you are right, this is more or less preparation for the followup ticket where the actual matching algorithm will be created.